MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -9,6 +9,9 @@ protocol FeedViewControllerDelegate {
 }
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+	
+	@IBOutlet weak var errorView: ErrorView?
```

Comment:
This property doesn't need to be `weak`. Also, the setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) var errorView: ErrorView?
```

Reply:
fixed